modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -108,17 +120,22 @@ curl "http://localhost/myResource" -u 1234:abcd
 
 Example DataWeave 2.0 expression to be used when configuring the policy:
 
-No DataWeave 2.0 expression needed to be specified when the policy is configured to use the Basic Authentication as the origin of the credentials.
+To use the Basic Authentication as the origin of the credentials, no DataWeave 2.0 expression needs to be specified when configuring the policy.
 
-== WWW Authenticate HTTP header
-When a HTTP request is performed against a protected resource, where the Client ID enforcement policy is applied, and client application credentials are invalid or unauthorized, the HTTP response includes a WWW Authenticate header with the following values:
+==== With WWW Authenticate HTTP header
```

Comment:
This is not another way of configuring the policy, it's an effect that is present no matter how you configure the policy. So, if you apply a client Id based policy and you don't provide the credentials when making the request, the response will have the 'WWW-Authenticate' header.

Reply:
Done! Thanks!